docs/flow/modin/experimental/engines/omnisci_on_ray/frame/index.rst

```diff
@@ -0,0 +1,45 @@
+OmniSciOnRay Frame Implementation
```

Comment:
Navigating to `Frame`,  `PartitionManager` and OmnisciServer` pages from this document I can't see their content/API. Any reasons?

Reply:
https://modin--2984.org.readthedocs.build/en/2984/flow/modin/experimental/engines/omnisci_on_ray/frame/index.html

Reply:
Looks like missing dependency during docs build is the reason:
```
WARNING: autodoc: failed to import class 'data.OmnisciOnRayFrame' from module 'modin.experimental.engines.omnisci_on_ray.frame'; the following exception was raised:
No module named 'dbe'
```

Reply:
Yes, probably, that causes the issue. What do we need to fix it?

Reply:
We need to install `pyomniscidbe` using `conda`.

Reply:
Or better to add a fake dependency [here](https://github.com/modin-project/modin/blob/master/docs/conf.py#L24) as we do for other similar cases.